internal_ws/ykbh/src/lib.rs

```diff
@@ -231,9 +231,9 @@ impl SIRInterpreter {
         &self.frames.last().unwrap().mem
     }
 
-    /// Inserts a pointer to the trace inputs into the `interp_step` frame.
-    pub fn set_trace_inputs(&mut self, tio: *mut u8) {
-        // The trace inputs live in $1
+    /// Inserts a pointer to the trace context into the `interp_step` frame.
+    pub fn set_trace_ctx(&mut self, tio: *mut u8) {
```

Comment:
Another `tio` here.

Reply:
These *should* be fixed in a later commit.

Reply:
Also a mention of "trace context" in the comment.

Reply:
ah yes. it's probably not worth me reviewing individual commits :P Sorry.